### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by callmecavs**
Just cleaned up a few things - commits are small and single-purpose for your review.

Housekeeping - Tommy Boy

### CODE REVIEW

File: javascript/markov.js [hunk 1]
```diff
@@ -75,16 +77,14 @@ class Markov {
...
 
   _random() {
-    if (this.words[key]) {
-      return this._getRandomElement(this.words[key])
-    } else {
-      return this._getRandomElement(this.words[this._getRandomKey()]);
-    }
+    return this.words[key]
+      ? this._getRandomElement(this.words[key])
+      : this._getRandomElement(this.words[this._getRandomKey()]);
```
Comment (jaxgeller) [inline javascript/markov.js]: This is going to be a problem, this._getRandomKey wont be defined

**Comment by callmecavs** [comment]
Meaning the 'this' context will be Array, not your object? Can always bind it

**Comment by jaxgeller** [comment]
Thats my bad, that method isnt being used.

**Comment by callmecavs** [comment]
i removed the unused private methods in the latest commit

**Pull request merged**